src/metrics/package.json

```diff
@@ -1,39 +1,38 @@
 {
   "name": "uproxy-metrics",
-  "version": "1.0.0",
-  "description": "Metrics server for uProxy",
-  "main": "metrics_server/server.js",
-  "dependencies": {
-    "csv-parse": "^1.1.7",
-    "express": "^4.14.0",
-    "gcloud": "^0.37.0"
-  },
+  "version": "1.0.0-beta",
+  "author": "uProxy",
+  "description": "Metrics systems for uProxy",
   "devDependencies": {
     "jasmine": "^2.4.1",
     "typescript": "^1.8.10",
     "typings": "^1.3.2"
   },
-  "//": "For start-datastore, see https://cloud.google.com/datastore/docs/tools/datastore-emulator",
+  "__comments": [
+    "TODO: Remove node_modules_hack once AppEngine stops stripping node_modules/"
```

Comment:
Is there any public issue or documentation you can link to here just so future developers can learn more?  If so might be good to include in the comment